data/json/monsters/zed_burned.json

```diff
@@ -3,24 +3,24 @@
     "id": "mon_zombie_child_scorched",
     "type": "MONSTER",
     "name": { "str": "zombie kinderling" },
-    "description": "A tiny charred body, jumping and kicked and flailing around in a mockery of playground exercise.  It does not need a face for you to feel bad about killing it.",
+    "description": "A tiny charred body - jumping, kicking, and flailing around in a mockery of playground exercise.  It does not need a face for you to feel bad about killing it.",
     "default_faction": "zombie",
     "bodytype": "human",
-    "species": [ "ZOMBIE" ],
+    "species": [ "ZOMBIE", "HUMAN" ],
     "diff": 2,
     "volume": "30000 ml",
     "weight": "40750 g",
-    "hp": 35,
-    "speed": 90,
+    "hp": 30,
+    "speed": 65,
```

Comment:
Comparison to other child zombies?

Reply:
this is comparing it to the base child zombie, the burned child is significantly faster for some reason, so i toned it down, the HP is meant to reflect on the increase in armor